### DISCUSSION TITLE
Unable to 'Enable Vercel DNS' for my .uk domain

### DISCUSSION THREAD

**Discussion by adamjdavies** [vercel/community#4753] [Help]
### Question

I have a .uk domain that I'm trying to set up with Vercel so that I can easily use wildcard domains. When adding the domain to the project, I'm not getting the options to configure the domain using Vercel's nameservers. Additionally, the domain is not showing up in the list of domains used by my account (i.e. https://vercel.com/dashboard/domains). 

I've just tried another domain that I own (a .xyz) domain and that's working fine. 

Any ideas what may be causing this?

**Comment by amyegan** [comment]
Did you already successfully assign the domain to your project? If you share the domain, I can look into it more.
